packages/ckeditor5-table-resize/LICENSE.md

```diff
@@ -15,4 +15,4 @@ Where not otherwise indicated, all CKEditor 5 table resize feature content is au
 Trademarks
 ----------
 
-**CKEditor** is a trademark of [CKSource](http://cksource.com) Frederico Knabben. All other brand and product names are trademarks, registered trademarks or service marks of their respective holders.
+**CKEditor** is a trademark of [CKSource](http://cksource.com) Holding sp. z o.o. All other brand and product names are trademarks, registered trademarks or service marks of their respective holders.
```

Comment:
This file will be removed anyway when `ckeditor5-table-resize` becomes the part of the `ckeditor5-table`, but I guess it doesn't harm to keep it updated until the last moment it is present in the project.